src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,27 +43,85 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean = {
+      if(b == true) false;
+      else
+        true;
+    }
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
-
-    def or(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean = {

```

Comment:
See comments for `not`.
The solution could be simplified, using 2 `if` branches instead of 3: suppose `left` is `true`. What value determines the result of the expression?